db/schema.rb

```diff
@@ -82,11 +82,14 @@
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.integer "role", default: 0
+    t.bigint "merchant_id"
+    t.index ["merchant_id"], name: "index_users_on_merchant_id"
```

Comment:
It might be odd to have merchant_id for all users.. I almost wonder if we should have a joins table, but wouldn't really make sense if a user has many merchants right?... Does that mean an entirely different table then? Employees table with user_id and merchant_id?